dynamiqs/utils/utils.py

```diff
@@ -53,7 +53,7 @@ def dag(x: ArrayLike) -> Array:
         Array([[1.-0.j, 0.-0.j]], dtype=complex64)
     """
     x = jnp.asarray(x)
-    return x.mT.conj()
+    return jnp.conjugate(x.mT)
```

Comment:
Exactly equivalent no? Also let's use the shorter `jnp.conj`.

Reply:
If we use the former and the recent fix to time array (#406), `dq.dag` works on time arrays, which is not the case here.

Reply:
Ok yes we can but it was before I had jitting issues but we can try to revert